### PR TITLE
Created aim and process of project

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
created process and aim of project in readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,26 @@
...
+---
+<h3 align="center">Workflow of the Project</h3>
+
+📌 Gathering information about how to target audience. Analyze user needs and preferences. <br>
+📌 Developing a project plan that includes project scope, timelines, milestones,resources, and budget. <br>
+📌 Based on user research, define the key features . Identify the user interface design, navigation, and user experience requirements. <br>
+📌 Develop it using the appropriate programming languages and frameworks. <br>
+📌 Testing  the app thoroughly to ensure its quality, usability, and performance. <br>
+📌 Releasing the app <br>
+📌 Continuously improve the app's features, design, and functionality based on user <br>
```
Comment (bhattabhi013) [inline README.md:25]: This whole section is not customized for the application.
I suggest you run the application on your device or watch the demo for writing the workflow.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,26 @@
...
+
+---
+
+<h3 align="center">Aim of the Project</h3>
+
+📌 We provide an engaging and interactive learning platform for young children to learn and understand nursery rhymes in a fun way <br>
+📌 We aim to incorporate features such 3D <br>
```
Comment (bhattabhi013) [inline README.md:32]: incomplete sentence.

**Review by bhattabhi013 — CHANGES_REQUESTED** [review]
Please do the suggested changes.

**Comment by Bharadwajshivam28** [comment]
> Please do the suggested changes.

Ok sir

**Comment by Bharadwajshivam28** [comment]
@bhattabhi013 I have improved contents sir please look to it

File: README.md [hunk 3]
```diff
@@ -16,22 +16,28 @@
...
 <h3 align="center">Workflow of the Project</h3>
 
-📌 Gathering information about how to target audience. Analyze user needs and preferences. <br>
-📌 Developing a project plan that includes project scope, timelines, milestones,resources, and budget. <br>
-📌 Based on user research, define the key features . Identify the user interface design, navigation, and user experience requirements. <br>
-📌 Develop it using the appropriate programming languages and frameworks. <br>
-📌 Testing  the app thoroughly to ensure its quality, usability, and performance. <br>
-📌 Releasing the app <br>
-📌 Continuously improve the app's features, design, and functionality based on user <br>
+📌 We have firstly identify the need of children like theri problems in studying <br>
```
Comment (bhattabhi013) [inline README.md:19]: their

File: README.md [hunk 4]
```diff
@@ -16,22 +16,28 @@
...
+📌 First of all when user will get to our app ,there will be Google authentication for login. <br>
+📌 User can login by their mail as it is easy to do <br>
+📌 Google authentication is very secured to use <br>
+📌 We have many rhymes for vast variety of choice <br>
+📌 One can read the text and one can also play the text, as for kids reading one is hard so they can play the text also <br>
```
Comment (bhattabhi013) [inline README.md:24]: One can read the text and also play with it using text-to-speech functionality, as for kids reading one is hard so they can listen to the text as well.

File: README.md [hunk 5]
```diff
@@ -16,22 +16,28 @@
...
-📌 We provide an engaging and interactive learning platform for young children to learn and understand nursery rhymes in a fun way <br>
-📌 We aim to incorporate features such 3D <br>
-📌 We aim cross-platform nature of the application aims to make it easily accessible to a wide range of users, regardless of their device or operating system. <br>
-📌 Overall, the aim of the project is to promote early childhood education and make learning fun and engaging for young children. <br>
+📌 Learning is easy and fun by Vritant <br>
+📌 Many features are there for best understanding of rhymes <br>
+📌 Vritant is purely made by new and unique ideas <br> 
+📌 We have made it in such a way that kids will not be bored while learning <br>
+📌 Vritant is easily accesible to all <br>
+📌 There is no paying of money to acess the app and features alsoas it is completely free <br>
```
Comment (bhattabhi013) [inline README.md:39]: There is no need to pay money as the app is free.

**Review by bhattabhi013 — CHANGES_REQUESTED** [review]
Please resolve

**Comment by Bharadwajshivam28** [comment]
> Please resolve

Ok sir

**Comment by Bharadwajshivam28** [comment]
@bhattabhi013 Corrected it sir

**Review by bhattabhi013 — APPROVED** [review]
LGTM

**Comment by Bharadwajshivam28** [comment]
please close the issue also sir

**Pull request merged**